ramls/inventory.raml

```diff
@@ -76,6 +76,102 @@ resourceTypes:
               body:
                 text/plain:
                   example: "Internal server error"
+      /mark-in-process:
+        post:
+          responses:
+            201:
```

Comment:
Why is this process responding with a 201 CREATED? What resource is being created during this process? 

As I understand it, an existing item is being changed, which would be better reflected by a 200 if the response includes the new item representation or a 204 if not.

Reply:
I'm sorry but I've been used already exists status API as template . I'm really didn't think about. I supposed that was agreed before. As for me 202 would be better.

Reply:
> I'm sorry but I've been used already exists status API as template

That is understandable. Given they are all in the same interface, maybe consistency is better at this point.

It seems I missed this in the original code review :-/ 

@bohdan-suprun can you remember why 201 was chosen rather than 200?

Reply:
@marcjohnson-kint, I don't remember also, maybe by a mistake. I agree that 200 works better here.

Thanks.

Reply:
Do we want to go with consistent use of 201 across all of them or 200 for the new ones and try to remember changing the older one when we can?

Reply:
@marcjohnson-kint, @bohdan-suprun  So we need to make changes for success code response for all statuses API from 201 to 200, right? How it would affect existing UI and how we should to notify UI Team about it before make changes?

Reply:
It worth asking @mkuklis or @zburke about this, I thought they used a component for the menu option, maybe it will be difficult to expect different statuses in the shared component.

My preference will be to change it for the new ones and then update existing.

Reply:
@bohdan-suprun, @AnShapovalov, @marcjohnson-kint I'm a bit out of context with this but if we are talking about checking the status responses the UI should work the same with `201` or `200`.

Reply:
Thanks @mkuklis. I'd suggest change statuses for the new APIs. Does it work for you @marcjohnson-kintt, @AnShapovalov?

Reply:
@bohdan-suprun That is fine by me